packages/browser-agent-core/common/harvest/harvest.js

```diff
@@ -155,7 +155,7 @@ export class Harvest extends SharedContext {
     return ([
       '?a=' + info.applicationID,
       encodeParam('sa', (info.sa ? '' + info.sa : '')),
-      encodeParam('v', VERSION),
+      encodeParam('v', `${VERSION}.${SUBVERSION}`),
```

Comment:
VERSION is being used in metrics instrument too; does that need to be changed? Can we rename this scheme to `{MAJOR}.{MINOR}.{PATCH}` per semantic versioning standard (naming)? Can you combine them to export `= {VERSION}` so we don't have to reconstruct them per caller?

Reply:
good catch.  Yes to both